lib/components/gene/__tests__/__snapshots__/about-test.js.snap

```diff
@@ -0,0 +1,197 @@
+exports[`test renders correctly 1`] = `
+<View>
+  <View>
+    <View
+      style={
+        Object {
+          "marginLeft": 0,
+          "marginRight": 0,
+        }
+      }>
+      <Text
+        accessible={true}
+        allowFontScaling={true}
+        ellipsizeMode="tail"
+        numberOfLines={0}
+        style={
+          Array [
+            Object {
+              "fontSize": 17,
+            },
+            Object {
+              "fontSize": 16,
+              "lineHeight": 20,
+              "marginBottom": 20,
+            },
+            Object {
+              "fontFamily": "AGaramondPro-Regular",
+            },
+          ]
+        }>
+        This is some example text
+      </Text>
```

Comment:
This is just me taking a screenshot to give you a reference at what the snapshot represents, you don't really need to do this. This is the real thing, nothing to do with the test itself.